src/builder.rs

```diff
@@ -387,11 +387,11 @@ impl DbcBuilder {
         self
     }
 
-    /// Build the output DBCs
+    /// Build the output DBCs, verifying the transaction and spentproofs.
     ///
     /// see TransactionVerifier::verify() for a description of
     /// verifier requirements.
-    pub fn build<K: KeyManager>(
+    pub fn verify_and_build<K: KeyManager>(
```

Comment:
I can do that, but there are no users of the API so....

Reply:
right, I thought it was about backward compatibility because that would mean any old user of the api could still assume that the semantic of the `build` API  implies verification, if there is no backward compatibility issue, and there is no really any users yet of this, then we can define/redefine the semantic of each API, including the `build`, if we simply define the `build` is just build then there is no need for making explicit, what I mean is `build` shouldn't per se necessarily mean there is a verification involved to anybody unless documented as that, that was how I was reasoning about it.